models/azure/storage_account_primary_endpoints.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2022 Specter Ops, Inc.
+//
+// This file is part of AzureHound.
+//
+// AzureHound is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// AzureHound is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+package azure
+
+type SAPrimaryEndpoints struct {
```

Comment:
When possible use the canonical Azure names for structs to maintain as much of 1-1 mapping with the Azure docs. In this case I think we should use `Endpoints`.

Reply:
Modified